ocs_ci/ocs/node.py

```diff
@@ -1073,3 +1073,64 @@ def generate_node_names_for_vsphere(count, prefix="compute-"):
         f"{prefix}{current_compute_suffix + node_count}"
         for node_count in range(1, count + 1)
     ]
+
+
+def check_taint_on_nodes(taint=None):
+    """
+    Function to check for particular taint on nodes
+
+    Args:
+        taint (str): The taint to check on nodes
+
+    """
+    taint = taint if taint else "node.ocs.openshift.io/storage=true:NoSchedule"
+    ocs_nodes = get_ocs_nodes()
+    flag = -1
+    for node_obj in ocs_nodes:
+        if node_obj.get().get("spec").get("taints"):
+            if node_obj.get().get("spec").get("taints")[0].get("key") in taint:
+                log.info(f"Node {node_obj.name} has taint {taint}")
+                flag = 1
+        else:
+            flag = 0
+    return bool(flag)
```

Comment:
this logic will return True even if only 1 node has the particular taint and the rest don't.
Is it good enough?